dags/bgbb.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.moz_databricks import MozDatabricksSubmitRunOperator
+from utils.mozetl import mozetl_envvar
+
+default_args = {
+    "owner": "wbeard@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime(2019, 2, 22),
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "wbeard@mozilla.com",
+        "amiyaguchi@mozilla.com",
+    ],
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+# run on the 8th hour of the 1st day of the month
+dag = DAG("bgbb_fit", default_args=default_args, schedule_interval="0 8 1 * *")
+
+
+bgbb_fit = MozDatabricksSubmitRunOperator(
```

Comment:
I think a comment here saying that this job uses clients_daily would be helpful (it'd be better if we could surface it in the airflow UI, but I'm not sure there's an easy way to do that)

Reply:
I think a dummy operator could work here, for documentation purposes.